packages/openapi-generator/package.json

```diff
@@ -15,31 +15,21 @@
   "main": "./dist/index.js",
   "types": "./dist/index.d.ts",
   "bin": {
-    "sap-cloud-sdk-rest": "./bin/run"
+    "generate-openapi-client": "./bin/run"
```

Comment:
This is the `sap-cloud-sdk` for the CLI so the command the generate will be called: `generate-openapi-client generate-openapi-client. ...` if this is the case perhaps you can give the comannd a shorter name or this bin.